CRM/Mailing/Transactional.php

```diff
@@ -297,6 +297,7 @@ public function verpify(&$params, $setReturnPath = TRUE) {
         'job_id' => $job_id,
         'email_id' => $email_id,
         'contact_id' => $contact_id,
+        'mailing_id' => $mailing_id,
```

Comment:
Cool and this is populated from result of call to getMailingIds which looks like it ensures that the mailing id is not null. Nice this was much easier than I was expecting!